tests/test_auth.py

```diff
@@ -204,6 +204,23 @@ def unsupported_callback(request):
         assert response.status_code == 401
         assert len(responses.calls) == 1
 
+    @responses.activate
+    def test_retry_get_token(self):
+        url = 'https://registry.example.com/v2/fedora/tags/list'
+
+        responses.add(responses.GET, BEARER_REALM_URL, status=503, json='unavailable')
+        responses.add(responses.GET, BEARER_REALM_URL, json={'token': BEARER_TOKEN})
+
+        responses.add_callback(responses.GET, url, bearer_unauthorized_callback)
+        responses.add_callback(responses.GET, url, bearer_success_callback)
+
+        auth = HTTPBearerAuth()
+
+        response = requests.get(url, auth=auth)
+        assert response.json() == 'success'
+        assert response.status_code == 200
+        assert len(responses.calls) == 4
+
```

Comment:
Test does not work, I suspect that `responses` does not support retries.

Does this change need a test? If so, does anyone have a good way to mock responses for retries?

Reply:
https://github.com/getsentry/responses/issues/135  :(

Probably not

Reply:
Ok, removed test

Reply:
Do we care enough about this to use something like 'httpretty' as in e.g. https://github.com/getsentry/responses/issues/135#issuecomment-644384328

Reply:
IMO the test is not so important that I would add an extra tests dependency, it would essentially be testing 3rd party functionality anyway (retries with `requests` + `urllib3`)